src/api/admin.api.js

```diff
@@ -0,0 +1,44 @@
+import { axiosInstance } from "./http";
+
+class AdminService {
+  async login(credentials) {
+    const admin = await axiosInstance
+      .post('admin/auth/sign-in/', credentials)
+      .then(data => data.data) 
+
+    if (admin?.token) {
+      localStorage.setItem(
+        'token',
+        admin.token
+      );
+    }  
+    return admin;  
+  }
+
+  async getVolunteersRequests() {
+    const requests = await axiosInstance
```

Comment:
return it directly without await
```suggestion
    return axiosInstance
```